docs/proposals/multi-node/README.md

```diff
@@ -732,7 +791,7 @@ Mark the status condition type `AllMembersReady` to `false`.
     - Ready: `r` where `r > n/2`
     - Members `NotReady` for long enough to be evicted, i.e. `lastTransitionTime > notReadyGracePeriod`: `nr` where `0 < nr < n/2`
     - Members with readiness status `Unknown` long enough to be considered `NotReady`, i.e. `lastTransitionTime > unknownGracePeriod`: `u` where `0 < u < n/2`
-    - Members with heartbeat stale enough to be considered as of `Unknown` readiness status, i.e. `lastHeartbeatTime > heartbeatGracePeriod`: `h` where `0 < h < n/2`
+    - Members with expired `lease`: N/A
```

Comment:
Hm, why not as before?
```
`h` where `0 < h < n/2` 
```

Reply:
The `lease` and member entries being out of sync was already addressed in [2. Member status is out of sync with their lease](https://github.com/gardener/etcd-druid/pull/207/files#diff-7918fb156cb729ac55570b3318ff8e792f008e5fe087f0a7c7a36db71374411bR694-R722). So, I thought adding more of the same here might be repetitive.

Reply:
Okay thanks. Thought, I missed something because of that change.